multimodal/src/autogluon/multimodal/models/ner_text.py

```diff
@@ -164,11 +164,11 @@ def forward(
                 attention_mask=text_masks,
             )
 
-        sequence_output = outputs.last_hidden_state
+        sequence_output = outputs[0]  # last_hidden_state
```

Comment:
How about using `last_hidden_state = outputs[0]` to be consistent with other models?

Reply:
Agree. changed sequence_output -> last_hidden_state.